### PR TITLE
Simple product duplication for June updates

### PR DESCRIPTION
**Pull request by NolwennP**
TODO:
- [x] Screenshots

### CODE REVIEW

File: content/updates/2020-06/1-duplicate-a-product.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# New: duplicate a product
+::: meta-data type="New" features="Enrichment, Product creation" available="early-June" link-to-doc="../articles/duplicate-a-product.html"
+:::
+
+Want to easily create a new product based on another? You can now duplicate a simple product in just one click! What a time-saver! :wink:
```
Comment (LaureBro) [inline content/updates/2020-06/1-duplicate-a-product.md:5]:
```suggestion
Do you want to easily create a new product from another? You can now duplicate a simple product in your user interface with one single click! What a time-saver! :wink:  
```

File: content/updates/2020-06/1-duplicate-a-product.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# New: duplicate a product
+::: meta-data type="New" features="Enrichment, Product creation" available="early-June" link-to-doc="../articles/duplicate-a-product.html"
+:::
+
+Want to easily create a new product based on another? You can now duplicate a simple product in just one click! What a time-saver! :wink:  
+You can find this new action in the `...` button of the product edit form of each one of your products.
```
Comment (LaureBro) [inline content/updates/2020-06/1-duplicate-a-product.md:6]:
```suggestion
Find this new action under the `...` button at the top right corner of each of your product edit forms. 
```

File: content/updates/2020-06/1-duplicate-a-product.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# New: duplicate a product
+::: meta-data type="New" features="Enrichment, Product creation" available="early-June" link-to-doc="../articles/duplicate-a-product.html"
+:::
+
+Want to easily create a new product based on another? You can now duplicate a simple product in just one click! What a time-saver! :wink:  
+You can find this new action in the `...` button of the product edit form of each one of your products. 
+
+There is a permission to hide this action, if you don't want some users to be able to duplicate products.
```
Comment (LaureBro) [inline content/updates/2020-06/1-duplicate-a-product.md:8]: This new feature also comes with a dedicated permission, so you can decide which group(s) of users can perform this action.

Comment (NolwennP) [reply content/updates/2020-06/1-duplicate-a-product.md:8]:
```suggestion
This new feature also comes with a dedicated permission, so you can decide which group(s) of users can perform this action.
```

File: content/updates/2020-06/1-duplicate-a-product.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# New: duplicate a product
+::: meta-data type="New" features="Enrichment, Product creation" available="early-June" link-to-doc="../articles/duplicate-a-product.html"
+:::
+
+Do you want to easily create a new product from another? You can now duplicate a simple product in your user interface with one single click! What a time-saver! :wink:
```
Comment (camillebourjade) [inline content/updates/2020-06/1-duplicate-a-product.md:5]:
```suggestion
You want to easily create a new product based on an existing one? You can now duplicate a simple product in your user interface in a single click! What a time-saver! :wink:  
```

File: content/updates/2020-06/1-duplicate-a-product.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# New: duplicate a product
+::: meta-data type="New" features="Enrichment, Product creation" available="early-June" link-to-doc="../articles/duplicate-a-product.html"
+:::
+
+Do you want to easily create a new product from another? You can now duplicate a simple product in your user interface with one single click! What a time-saver! :wink:  
+Find this new action under the `...` button at the top right corner of each of your product edit forms.
```
Comment (camillebourjade) [inline content/updates/2020-06/1-duplicate-a-product.md:6]:
```suggestion
Find this new action under the `...` button in the top right corner of each of your product edit forms. 
```

Comment (camillebourjade) [reply content/updates/2020-06/1-duplicate-a-product.md:6]: I'm not sure about the 'each of your pef' -> would it be accurate to say in the top right corner of the pef (singular)? if It works, it'd be lighter!

File: content/updates/2020-06/1-duplicate-a-product.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# New: duplicate a product
+::: meta-data type="New" features="Enrichment, Product creation" available="early-June" link-to-doc="../articles/duplicate-a-product.html"
+:::
+
+Do you want to easily create a new product from another? You can now duplicate a simple product in your user interface with one single click! What a time-saver! :wink:  
+Find this new action under the `...` button at the top right corner of each of your product edit forms. 
+
+This new feature also comes with a dedicated permission, so you can decide which group(s) of users can perform this action.
```
Comment (camillebourjade) [inline content/updates/2020-06/1-duplicate-a-product.md:8]:
```suggestion
This new feature also comes with dedicated permission, so you can decide which group(s) of users can perform this action.
```

File: content/updates/2020-06/1-duplicate-a-product.md (hunk 5 above)
Comment (NolwennP) [reply content/updates/2020-06/1-duplicate-a-product.md:6]: Yes, that'd work!

**Pull request merged**